README-CONAN.md

```diff
@@ -1,108 +1,193 @@
+
+To build Exiv2 with conan, you will also need to install CMake.  For more information see [README-CMAKE](README-CMAKE)
+
+<name id="TOC"></a>
+----
+### T A B L E &nbsp;&nbsp;&nbsp; O&nbsp;&nbsp;F &nbsp;&nbsp;&nbsp; C O N T E N T S
+
+1. [Step by Step Guide](#1)
+    1. [Install conan](#1-1)
+    2. [Test conan installation](#1-2)
+    3. [Create a build directory](#1-3)
+    4. [Build dependencies and install conan artefacts in your build directory](#1-4)
+    5. [Execute cmake to generate build files for your environment:](#1-5)
+2. [Platform Notes](#2)
+    1. [Linux Notes](#2-1)
+    2. [Visual Studio Notes](#2-2)
+    3. [Cygwin Notes](#2-3)
+    4. [MinGW Notes](#2-4)
+3. [Conan Architecture](#3)
+    1. [conanfile.py](#3-1)
+    2. [Conan Recipes](#3-2)
+    3. [Conan server search path](#3-3)
+    4. [Configuring conan on your machine](#3-4)
+4. [Building Exiv2 with Adobe XMPsdk 2016](#4)
+    1. [Add a remote directory to conan's recipe search path](#4-1)
+    2. [Build dependencies and install conan artefacts in your build directory](#4-2)
+    3. [Execute cmake to generate build files for your environment](#4-3)
+    4. [Build Exiv2 and link Adobe XMPsdk library](#4-4)
+
+<name id="1"></a>
+----
+# 1 Step by Step Guide
+
+<name id="1-1"></a>
+##### 1.1) </a>Install conan:
 
+```bash
+$ pip install conan
+```
+For other installation methods (brew, installers, from sources), visit this [link]([install
+conan](http://docs.conan.io/en/latest/installation.html)).
 
-## Main concepts
+To upgrade the version of conan:
 
-Even when the following instructions will allow you to use conan directly, we strongly recommend to take a look to their
-"Getting started" [page](http://docs.conan.io/en/latest/getting_started.html), to better understand the main concepts.
+```bash
+$ pip install conan --upgrade
+```
 
-In the root level of the **Exiv2** repository, there is a file named `conanfile.py` that defines the C/C++ dependencies
-of the project. One of the dependencies is **Expat**, and it is stated like this:
+<name id="1-2"></a>
+##### 1.2) Test conan installation
 
-```python
-self.requires('Expat/2.2.5@pix4d/stable')
+```bash
+$ conan --version
+Conan version 1.4.1
 ```
 
-The syntax stands for: `Library/version@user/channel`.
+<name id="1-3"></a>
+##### 1.3) Create a build directory<name id="1-3"></a>
 
-Conan will search for these dependencies in a list of remotes. In those servers there are recipes (the python code that
-explains how to build a library from its sources) and binary packages (different versions of the library for several
-combinations of configurations: Platform/Compiler/Settings). If there is already a packages for your configuration,
-conan will fetch them. Otherwise, it will compile the libraries for you by using the instructions provided in the
-recipe. To better illustrate this idea, here we show a list of packages that is returned by the command `conan search`
+Create a build directory and will run the conan commands:
```

Comment:
I think the "will" does not belong into this sentence.

Reply:
I agree. This is a typo.  I've removed the word `will`.